app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala

```diff
@@ -217,7 +217,7 @@ class BitcoinSServerMain(override val serverArgParser: ServerArgParser)(implicit
       nodeConf: NodeAppConfig,
       ec: ExecutionContext): Future[NodeCallbacks] = {
     lazy val onTx: OnTxReceived = { tx =>
-      logger.info("Receiving transaction as a callback")
+      logger.info(s"Receiving transaction txid=${tx.txId} as a callback")
```

Comment:
So another nit, but a lesson worth learning: 

we have two methods for `txId`. 

There is `txId` (which you are using) and `txIdBE`. The BE stands for big endian. Satoshi Nakamoto made the bitcoin protocol use little endian byte encoding rather than the common big endian standard. 

To confuse you even more, all transaction ids that are used on block explorer or returned from rpc calls are big endian. Thus, when logging you shoudl always use `txIdBE` so you can easily search for the tx on a block explorer.

https://bitcoin.stackexchange.com/a/41059/20835

Reply:
Oh, that was pretty informative! I'll keep that in mind the next time I'm logging. I'll change txId to txIdBE here also